.github/workflows/busola-backend-build.yml

```diff
@@ -25,6 +31,4 @@ jobs:
     with:
       name: busola-backend
       dockerfile: Dockerfile
-      context: backend
-      export-tags: true
-      tags: latest
```

Comment:
make sure that we can remove this

Reply:
`export-tags` is not needed in my opinion. According to docs:
```
description: Export parsed tags as build-args into dockerfile. Each tag will have format TAG_x, where x is the tag name passed along with the tag
```
I think that it's unnecessary complication and it hides the build logic.